src/MSALWrapper/AuthFlow/AuthFlowFactory.cs

```diff
@@ -45,6 +45,13 @@ public class AuthFlowFactory
             // as it sets the order in which auth flows will be attempted.
             List<IAuthFlow> flows = new List<IAuthFlow>();
 
+            // We try IWA as the first auth flow as it works for any Windows version
+            // and tries to auth silently if it finds an account in the cache.
+            if (authMode.IsIWA())

```

Comment:
IWA only makes sense on Windows on below we are also checking if Windows for WAM. Do we need to check for windows here> or is the `IsIWA()` platform aware?

Reply:
Added platformUtils.IsWindows() flag check for IWA. IsIWA() did check for windows platform but we should check it before queuing as well just like we do for Broker. Good catch!